oggm/sandbox/run_alaska.py

```diff
@@ -220,18 +249,19 @@
     for gdir in gdirs:
         gdir.inversion_calving_rate = 0
 
+    cfg.PARAMS['correct_for_neg_flux'] = False
+    cfg.PARAMS['filter_for_neg_flux'] = False
```

Comment:
Yes, but be careful to use the very last oggm version here (after @TimoRoth last PR - without it these parameters might be ignored)

Reply:
I dont understand this comment? 

And also about this setting I keep think I will have to do separate runs. One for land terminating and one for tidewater glaciers. I think that there should be a distinction inside the function between non-tidewater and tidewater. So, we don't have to worry about making separate runs or making extra ifs in run_alaska.py

Reply:
Disregard my comment - if you updated oggm after timo's last PR, you are good